gufe/settings/settings.py

```diff
@@ -0,0 +1,141 @@
+"""
+Convenience functions for working with Settings
+"""
+
+import json
+import warnings
+from pathlib import Path
+
+from openff.toolkit.typing.engines.smirnoff import ForceField
+from openff.toolkit.utils.exceptions import SMIRNOFFParseError
+
+from .models import (AngleSettings, BondSettings, ElectrostaticSettings,
+                     ForcefieldSettings, GBSASettings, ImproperTorsionSettings,
+                     ProperTorsionSettings, Settings, ThermoSettings,
+                     VdWSettings)
+
+
+def ff_settings_from_offxml(force_field) -> ForcefieldSettings:
+    """
+    Creates a ForcefieldSettings object from an open forcefield ForceField object.
+    """
+    vdW = VdWSettings(
+        cutoff=force_field.get_parameter_handler("vdW").cutoff.magnitude,
```

Comment:
would it make sense to store force_field.get_parameter_handler("vdW") in a variable?

Reply:
I'm not sure why, once I get units in this line will be:
`cutoff=force_field.get_parameter_handler("vdW").cutoff,`